src/components/participants-list/participants-list.tsx

```diff
@@ -0,0 +1,43 @@
+import { Participant } from "@/app/events/[event]/participants/page";
+
+interface ParticipantListProps {
+  participants: Participant[];
+}
+
+export function ParticipantsList({ participants }: ParticipantListProps) {
+  return (
+    participants.length && (
+      <ul className="inline-flex mt-[12px]">
+        {participants.map((participant, index) => (
+          <li
+            className="flex flex-col justify-center ml-2"
+            key={participant.id}
+          >
+            <div
+              className={`w-[69px] h-[69px] border-2 border-primary-100 rounded-full ${
+                index !== participants.length - 1
```

Comment:
This is hard to read, we could move it to a dedicated variable/expression.
```suggestion
const <pick-a-good-name> = participants.length - 1 ? "<classes-here>" : "<classes-here>"
```

Reply:
Done